Update ja.xliff

kenroy opened the pull request:
Tuesday night translations. Go Oilers!

cliss reviewed (changes requested):
Thanks! It seems like you [perhaps accidentally] renamed the parent directory from `ja.xcloc` to `ja-JP.xcloc`; would you mind renaming it back please? Once you do that I'll merge. :)

kenroy commented:
Changed back. I had changed it deliberately thinking that I should probably name it the way we normally refer to Japanese. I noticed that French., Portuguese and Spanish had named the directories that way.

-Kenroy


> On Jun 23, 2024, at 11:07 AM, Casey Liss ***@***.***> wrote:
> 
> 
> @cliss requested changes on this pull request.
> 
> Thanks! It seems like you [perhaps accidentally] renamed the parent directory from ja.xcloc to ja-JP.xcloc; would you mind renaming it back please? Once you do that I'll merge. :)
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/cliss/callsheet-localizations/pull/115#pullrequestreview-2134308154>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAO5WLWI2SJB22HTS7F6ZNTZI3QBRAVCNFSM6AAAAABJRGWFLWVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDCMZUGMYDQMJVGQ>.
> You are receiving this because you authored the thread.
>

cliss reviewed (approved):
Thanks! I get the logic behind the change, but a lot of the localization stuff on my/Apple's side is based on convention. So I gotta stick with what Apple comes up with. :)
